src/components/new-request/search.vue

```diff
@@ -434,8 +428,9 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin, Sear
 
     // Conditional for Continuation In Flow.
     if (this.isContinuationIn) {
-      if (this.getEntityTypeCd && this.isNamedCompany) return true
+      if (this.getEntityTypeCd && !this.isNumberedEntityType) return true
       if (this.getEntityTypeCd && !this.isNumberedEntityType && this.isSociety) return true
+      if (this.getEntityTypeCd && this.isNamedCompany) return true
```

Comment:
Yup. Amalgamations had several bugs related to this (that I fixed), especially when changing between entity types.